MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -137,6 +137,13 @@
 #include <setjmp.h>
 #include <unistd.h>
 
+jmp_buf jbAnagram;
+
+void *memset(void *s : byte_count(n), int c, size_t n) : bounds(s, (char *) s + n);
```

Comment:
Yes. memset is declared in stdlib_checked.h
But, in this application, it defines its own `random `function but stdlib.h also has `random`
To avoid redefinition error of `random `function, I did not include `stdlib_checked.h`